tests/quart/test_graphiqlview.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+from quart import Quart, url_for
+from quart.testing import QuartClient
+
+from .app import create_app
+
+
+@pytest.fixture
+def app() -> Quart:
+    # import app factory pattern
+    app = create_app(graphiql=True)
+
+    # pushes an application context manually
+    # ctx = app.app_context()
+    # await ctx.push()
+    return app
+
+
+@pytest.fixture
+def client(app: Quart) -> QuartClient:
+    return app.test_client()
+
+
+@pytest.mark.asyncio
+async def test_graphiql_is_enabled(app: Quart, client: QuartClient):
+    async with app.test_request_context("/", method="GET"):
```

Comment:
Should be changed for Quart 0.6.15 to be like: https://gitlab.com/pgjones/quart/-/blob/0.6.15/docs/testing.rst